packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -95,6 +95,34 @@ export async function copyProjectTemplateAndReplace(
     namespace = namespace.split(/[.:]+/).map(pascalCase).join('.');
   }
 
+  // Checking if we're overwriting an existing project and re-uses their projectGUID
+  const projectPathVcx = path.join(
+    windowsDir,
+    newProjectName,
+    newProjectName + '.vcxproj',
+  );
+  const projectPathCs = path.join(
+    windowsDir,
+    newProjectName,
+    newProjectName + '.csproj',
+  );
+  let reuseProjectGuid;
+  if (fs.existsSync(projectPathVcx)) {
+    console.log('Found exisitng C++ project, using old projectGUID');
```

Comment:
nit: a log statement below ends with a period. We should be consistent between statements.